Modules/Setting/Support/Settings.php

```diff
@@ -32,16 +32,16 @@ public function get($name, $locale = null, $default = null)
         $defaultFromConfig = $this->getDefaultFromConfigFor($name);
 
         $setting = $this->setting->findByName($name);
-        if (! $setting) {
+        if ($setting === null) {
             return is_null($default) ? $defaultFromConfig : $default;
         }
 
         if ($setting->isTranslatable) {
             if ($setting->hasTranslation($locale)) {
-                return empty($setting->translate($locale)->value) ? $defaultFromConfig : $setting->translate($locale)->value;
+                return trim($setting->translate($locale)->value) === '' ? $defaultFromConfig : $setting->translate($locale)->value;
             }
         } else {
-            return $setting->plainValue === null ? $defaultFromConfig : $setting->plainValue;
+            return trim($setting->plainValue) === '' ? $defaultFromConfig : $setting->plainValue;
```

Comment:
I wasn't completely sure about this change here... It all depends on if the [`ConvertEmptyStringsToNull`](https://github.com/laravel/framework/blob/5.5/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php) middleware is being used or not.